src/test/java/ro/utcn/sd/mdantonio/StackUnderflow/resources/db/migration/V1__Deployment_schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE IF NOT EXISTS underflowuser(
```

Comment:
Why did you duplicate this migration? :) Also the location of this whole resources folder is wrong -> it should be located directly under the `test` folder (next to the `java` folder). Check out the Maven Standard Directory Structure: https://www.baeldung.com/maven-directory-structure

Reply:
I was uncertain if it was also applied when running the tests and made a preemptive copy-paste there.